### PR TITLE
Approve information Sharing CSS

### PR DESCRIPTION
**Pull request by meodai**
Screenshot 2023-12-14 at 18 22 29

- Replicate the VC screen as well as possible using existing components 
- Add the concept of ".t-formatted" for reformatted HTML text. In this case Markdown
- Slightly modify the dapps Explorer component to be useful for origins 
- Add a `.c-card--consent` class

<hr/><details><summary>🟡 Some screens were changed</summary></details>

### CODE REVIEW

**Comment by meodai** [comment]
@frederikrothenberger I don't understand why the usePin workflow is failing :(

**Comment by frederikrothenberger** [comment]
> @frederikrothenberger I don't understand why the usePin workflow is failing :(

It seems to have been a viewport issue on mobile sized chrome. I have pushed a fix to this PR.

File: src/frontend/src/flows/verifiableCredentials/allowCredentials.ts [hunk 1]
```diff
@@ -8,12 +8,33 @@ import { html, TemplateResult } from "lit-html";
...
+import { getDapps, KnownDapp } from "../dappsExplorer/dapps";
+import { dappTemplate } from "../dappsExplorer/index";
+
 import DOMPurify from "dompurify";
 
 import copyJson from "./allowCredentials.json";
 
 /* A screen prompting the user to allow (or cancel) issuing verified
  * credentials */
+const getOrigin = (
```
Comment (frederikrothenberger) [inline src/frontend/src/flows/verifiableCredentials/allowCredentials.ts:20]: The comment above should be move to the template below, no?

File: src/frontend/src/flows/verifiableCredentials/allowCredentials.ts [hunk 2]
```diff
@@ -8,12 +8,33 @@ import { html, TemplateResult } from "lit-html";
...
+
+  if (!foundDapp) {
+    foundDapp = new KnownDapp({
+      name: "Unknown Dapp",
+      website: origin,
+      logo: "unwnowndapp.png",
+      oneLiner: oneLiner,
+    });
+  }
+  foundDapp.oneLiner = oneLiner;
```
Comment (frederikrothenberger) [inline src/frontend/src/flows/verifiableCredentials/allowCredentials.ts:35]: Nit: I think it is slightly hacky to override the `oneLiner` with the `copy` string on the `KnownDapp`.
What about overriding the value in the call to `dappsTemplate`?

**Review by frederikrothenberger — APPROVED** [review]
Two minor comments, otherwise looks good to me.

Disclaimer, I did not look at the CSS.

File: src/frontend/src/flows/verifiableCredentials/allowCredentials.ts (hunk 1 above)
Comment (meodai) [reply src/frontend/src/flows/verifiableCredentials/allowCredentials.ts:20]: yes of course! Let me fix that

File: src/frontend/src/flows/verifiableCredentials/allowCredentials.ts (hunk 2 above)
Comment (meodai) [reply src/frontend/src/flows/verifiableCredentials/allowCredentials.ts]: @frederikrothenberger you are right, I think its better now. let me know if this works for you

**Pull request merged**